src/pages/Detail/Modal/ModalShare.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+
+const ModalShare = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const [success, setSuccess] = useState(false);
+
+  const textInput = useRef();
+
+  const copy = () => {
+    const copyText = textInput.current;
+    copyText.select();
+    document.execCommand('copy');
+    setSuccess(true);
+  };
+
+  const closeModal = () => {
+    setOpenModal(false);
+    setSuccess(false);
+  };
+
+  return (
+    <AllContainer>
+      <BtnShare
+        onClick={() => {
+          setOpenModal(true);
+        }}
+        alt=""
+        src={process.env.PUBLIC_URL + 'images/icon_share.svg'}
+      />
+      {openModal && (
+        <ModalShareBg>
+          <ModalOverlay onClick={closeModal} />
+          <ModalShareContainer>
+            <ModalTop>
+              <button onClick={closeModal}>X</button>
+              <h1>공유하기</h1>
+              <h2>이 포지션과 어울리는 사람을 알고 있다면, 공유해주세요!</h2>
+            </ModalTop>
+            <hr />
+            <div onClick={copy}>
+              <p>링크 공유</p>
+              <input
+                ref={textInput}
+                type="text"
+                value="http://www.naver.com"
+                readOnly
+              />
+              <button>복사</button>
+              {success === true && <Success>복사되었습니다</Success>}
```

Comment:
success자체가 이미 boolean인데 ===으로 비교할 필요가 있을까요?
```suggestion
              {success && <Success>복사되었습니다</Success>}
```

Reply:
수정했습니다!